ui/app/AppLayouts/Browser/BrowserLayout.qml

```diff
@@ -207,7 +207,7 @@ Rectangle {
                         toastMessage.source = "../../img/loading.svg"
                         toastMessage.iconColor = Style.current.primary
                         toastMessage.iconRotates = true
-                        toastMessage.link = `${_walletModel.etherscanLink}/${responseObj.result}`
+                        toastMessage.link = `${_walletModel.etherscanLink}/${responseObj.result.result}`
```

Comment:
That's a bit ugly. Is there a way we could return only the first result or it's necessary to send it like this?